dbt/dbt_nmckinney/dbt_project.yml

```diff
@@ -21,6 +21,6 @@ require-dbt-version: [">=1.0.0", "<2.0.0"]
 
 models:
   nmckinney_dev:
-      materialized: table
-      stage:
-        materialized: view
\ No newline at end of file
+    polygon_stock_data:
+      materialized: view
+      schema: nmckinney_dev
```

Comment:
Remove this as this might be causing the issue you sent to me in your screenshot